job_creator.php

```diff
@@ -21,16 +23,16 @@ class JobCreator
      */
     public function getInstallerVersion(): string
     {
-        $repo = explode('/', $this->githubRepository)[1];
+        $this->repoName = explode('/', $this->githubRepository)[1];
```

Comment:
This is needed so that we have access to the repo name in `createJob()` - and since we now have this property we should use it wherever else we need the repo name as well.